build.gradle

```diff
@@ -58,6 +58,8 @@ minecraft.version = "1.7.10-10.13.4.1614-1.7.10"
 minecraft {
     version = "1.7.10-10.13.4.1614-1.7.10"
     runDir = "$projectDir"
+	replaceIn "MainMod.java"
```

Comment:
I have 0 idea what this does.

Reply:
replaces the version definition in my mod/submodule

Reply:
https://github.com/bartimaeusnek/bartworks-code/blob/9cef3f2d33dc444afae119badfbdccc55415b30d/github/bartimaeusnek/bartworks/MainMod.java#L25

Reply:
Yupp, basically the same method you used for GT++ Australia, makes it easier to to keep the code seperated and your code untouched.